src/components/styles/displayBook.css

```diff
@@ -0,0 +1,132 @@
+.displayBook {
+  display: flex;
+  justify-content: space-between;
+  width: 94%;
+  height: 8.5em;
+  margin: 0 0.063em 0.938em 0;
+  padding: 2em 2.5em;
+  border-radius: 4px;
+  border: solid 1px #fff;
+  background-color: #fff;
+}
+
+.book-details {
+  width: 50%;
+}
+
+.progress {
+  width: 50%;
+  padding: 0 5em 0 10em;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.category-name {
+  color: rgb(12, 12, 12, 0.4);
+  font-family: 'Montserrat', sans-serif;
+  font-size: 1em;
+}
+
+.book-title {
+  font-family: 'Montserrat', sans-serif;
+  color: black;
+  font-size: 1.5em;
+}
+
+.book-author {
+  color: #4386bf;
+  font-size: 1em;
+  font-family: 'Montserrat', sans-serif;
+}
+
+.actions {
+  display: flex;
+  gap: 1em;
+  padding-bottom: 1em;
+}
+
+.comments {
+  color: #4386bf;
+  font-size: 1em;
+  font-family: 'Montserrat', sans-serif;
+}
+
+.remove-btn {
+  color: #4386bf;
+  font-size: 1em;
+  font-family: 'Montserrat', sans-serif;
+  border: none;
+  padding: 0;
+  background-color: transparent;
+  cursor: pointer;
+}
+
+.remove-btn:hover {
+  color: red;
+}
+
+.edit-btn {
+  color: #4386bf;
+  font-size: 1em;
+  font-family: 'Montserrat', sans-serif;
+}
+
+.progress-oval {
+  display: flex;
+  gap: 1em;
+}
+
+.oval {
+  border-radius: 50%;
+  width: 3.5em;
+  height: 3.5em;
+  border: 5px solid #379cf6;
+  border-left-color: #e8e8e8;
+  -webkit-transform: rotate(45deg);
+  transform: rotate(45deg);
+}
+
+.progress-percent {
+  display: flex;
+  flex-direction: column;
+}
+
+.percentage {
+  color: black;
+  font-size: 2em;
+  font-family: 'Montserrat', sans-serif;
+  display: flex;
```

Comment:
- [ ] `display: flex `is also repeating severally. You can add a `flex className` in your code and do a `display: flex `just once in your CSS.
**For example:**
_In your displayBook.js_
` <div className="displayBook flex">`

_In displayBook.css_
```
.flex {
      display: flex;
}
```
You can check and apply this to all repeating rules

Reply:
Your suggestions are not working. I have tried several times.